README.md

```diff
@@ -1,16 +1,38 @@
-THIS IS A PLACEHOLDER
+# Bety
+Bety is a Web-interface to the Biofuel Ecophysiological Traits and Yields Database (used by [PEcAn](https://pecanproject.github.io/) and [TERRA REF](https://terraref.org/))
 
-## Upgrading BETY
+## Prerequisites
+- [Kubernetes](https://kubernetes.io/)
+- [Helm3](https://helm.sh/)
+
+## Installation
+
+> Before installing chart, make sure you enable NCSA repository.
+
+```
+helm repo add ncsa https://opensource.ncsa.illinois.edu/charts/
+```
+
+To install the chart with the release name my-release:
+
+```
+helm install my-release ncsa/betydb
+```
+> **Tip**: List all releases using `helm list`
+
+The command deploys Betydb on the Kubernetes cluster in the default configuration.
 
 If the password is generated you will need to save this secret before you upgrade. You can do this using the following commands. **If you do not do this, you will not be able to retrieve the previous secrets**.
 
 ```
-BETY_PASSWORD=$(kubectl get secrets betydb -o json | jq -r '.data.betyPassword' | base64 -d)
-BETY_SECRETKEY=$(kubectl get secrets betydb -o json | jq -r '.data.secretKey' | base64 -d)
-POSTGRESQL_PASSWORD=$(kubectl get secrets betydb-postgresql -o json | jq -r '.data."postgresql-password"' | base64 -d)
+echo BETY_PASSWORD=$(kubectl get secrets betydb -o json | jq -r '.data.betyPassword' | base64 -d)
```

Comment:
do you want to add a note maybe, to make sure they run this in the right namespace?

I was also thinking we should updates `templates/NOTES.txt` to use:
```
kubectl get --namespace {{ .Release.Namespace }} secrets betydb -o json | jq -r '.data.betyPassword' | base64 -d
```